perf/CMakeLists.txt

```diff
@@ -15,19 +15,23 @@ separate_arguments(BENCH_CMD_SEPARATE UNIX_COMMAND ${BENCH_CMD})
 
 add_subdirectory(lua)
 
-find_package(benchmark QUIET)
-if (NOT ${benchmark_FOUND})
-    message(AUTHOR_WARNING "Google Benchmark library was not found")
-    set(MSG "Target test-c-perf is dummy, Google Benchmark library was not found")
-    add_custom_target(test-c-perf
-                      COMMAND ${CMAKE_COMMAND} -E cmake_echo_color --red ${MSG}
-                      COMMENT ${MSG}
-    )
-    add_custom_target(test-perf
-                      DEPENDS test-c-perf test-lua-perf
-                      COMMENT "Running performance tests"
-    )
-    return()
+if(NOT ENABLE_BUNDLED_LIBBENCHMARK)
+    find_package(benchmark QUIET)
+    if(NOT ${benchmark_FOUND})
```

Comment:
I suppose that

```
    if(NOT ${benchmark_FOUND})
...
    endif()
```

should not be under `if(NOT ENABLE_BUNDLED_LIBBENCHMARK)`, right?

I would instead make like following:

```
if(ENABLE_BUNDLED_LIBBENCHMARK)
    include(BuildBenchmark)
    add_dependencies(build_bundled_libs bundled-libbenchmark)
else()
    find_package(benchmark QUIET)
    set(BENCHMARK_LIBRARIES benchmark::benchmark)
endif()

if(NOT ${benchmark_FOUND}
        add_custom_target(test-c-perf
...
        add_custom_target(test-perf
...
endif()
```

Reply:
No, it should since we search and check the existence of the benchmark library when we don't want to use the built-in one.

Reply:
In a private conversation decided to left as is.